modules/openapi-generator/src/main/resources/typescript-angular/encoder.mustache

```diff
@@ -15,7 +15,8 @@ export class CustomHttpParameterCodec implements HttpParameterCodec {
         return encodeURIComponent(k);
     }
     encodeValue(v: string): string {
-        return encodeURIComponent(v);
+        return encodeURIComponent(v)
+          .replace(/%2C(ASC|DESC)$/i, ",$1");
```

Comment:
I dont think this should be here

Reply:
Well, see #4404: Optionally, there could be a sort direction (ASC, DESC) appended like in
ansatz,DESC. The comma (,) should not be urlencoded.

It is for this special case (but it's a bit hacky - should I obmit it?)

Reply:
I think this should not be done here

Reply:
Hhm, I have no idea where else to put this. Hence I reverted the change...